content/en/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -158,6 +158,10 @@ Follow the steps given below to update your Deployment:
     ```shell
     kubectl --record deployment.apps/nginx-deployment set image deployment.v1.apps/nginx-deployment nginx=nginx:1.9.1
     ```
+    or simply use the following command : 
+    ```shell
```

Comment:
pls pay attention to the indentation, the current indentation may not work.

Reply:
please @tengqm  review my changes after fixing the indentation, thanks :)